### DISCUSSION TITLE
error installing python/mpf on windows 10

### DISCUSSION THREAD

**Discussion by Paulbaptiste** [missionpinball/missionpinball#26] [Q&A]
I continue to get errors when attempting to install Python and update. Images attached.  This was running 3.9.13 version, also tried going down a version and same results.

I'm following the instructions listed.  Downloaded python, install python and this is what happens.  I'm uninstalled and reinstalled 10 times or more.  

1st image is after following the first 2 commands

pip install --user pipx 
python -m pipx ensurepath

2nd image is after attempting

pipx install "mpf[cli]" --pip-args="--pre" --verbose --include-deps
(can't get past this)

I had no trouble installing on my mac, but I can't get this to work on my windows 10 laptop.

Appreciate any guidance
![pipx step one](https://user-images.githubusercontent.com/118682530/202915505-d7537e71-3146-49c7-b9a1-845b0cfd12fd.png)
![pipx step two error](https://user-images.githubusercontent.com/118682530/202915509-08f56078-fa69-47fd-8cec-6a7426f40a89.png)

**Comment by Pucksnok** [comment]
I saw you got answer on this at Google group.
